### PR TITLE
docs: using the session to JWT feature

### PR DESCRIPTION
**Pull request by aeneasr**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: docs/identities/session-to-jwt-cors.mdx [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+# Convert Ory Sessions to JSON Web Tokens
+
+Ory offers a powerful session management system that is based on cookies. In some contexts however, relying on Cookies is not
+possible:
+
+- You need to make a cross-origin (CORS) request where including the Ory Session Cookie is not trivial or possible.
+- You need a JWT which represents a signed-in user.
+- You need to integrate with a third party such as
+  [Zendesk SSO JWTs](https://support.zendesk.com/hc/en-us/articles/4408845838874).
+- You want to reduce the amount of calls to Ory's APIs.
```
Comment (kmherrmann) [inline docs/identities/session-to-jwt-cors.mdx:16]:
```suggestion
- You want to reduce the number of calls to Ory's APIs.
```

File: docs/identities/session-to-jwt-cors.mdx [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+Next, we configure our Ory Network project's tokenizer templates. The key we choose here is `jwt_example_template1`. We supply
+that template with the base64-encoded files from above:
+
+```shell
+ory patch identity-config {project.id} \
+  --add '/session/whoami/tokenizer/templates/jwt_example_template1={"jwks_url":"base64://'$JWKS_B64_ENCODED'","claims_mapper_url":"base64://'$JSONNET_B64_ENCODED'","ttl":"10m"}' \
+  --format yaml
+```
+
+Great! Everything is set up! To
```
Comment (kmherrmann) [inline docs/identities/session-to-jwt-cors.mdx:62]: To... - something missing here?

Comment (aeneasr) [reply docs/identities/session-to-jwt-cors.mdx:62]: Fuck, I missed your review

Comment (aeneasr) [reply docs/identities/session-to-jwt-cors.mdx:62]: https://github.com/ory/docs/pull/1541

**Pull request merged**